elide-core/src/main/java/com/yahoo/elide/core/RequestScope.java

```diff
@@ -58,6 +66,9 @@
     private final MultipleFilterDialect filterDialect;
     private final Map<String, FilterExpression> expressionsByType;
 
+    @Getter private final Map<String, Long> checkStats;
+    @Getter private final Set<Triple<Class, Class, String>> expressionResultShortCircuit;
```

Comment:
You could be more specific about Class here - like Class extends Annotation for one of them.